acquia/acquia_cms
# ACMS-625: Change the default acquia_cms theme to olivero

**Motivation**
Change the default acquia_cms theme to olivero
Fixes #NNN

**Proposed changes**
Change the default acquia_cms theme to olivero.

Provide search facet config for Olivero

 

**Testing steps**
Running site install and check how the theme behaves, and how the filters of search page are working.
